versions.props

```diff
@@ -1,5 +1,5 @@
 org.slf4j:* = 1.7.25
-org.apache.avro:avro = 1.9.2
+org.apache.avro:avro = 1.10.1
```

Comment:
Hi, @Fokko .
Could you update this PR with `1.10.2`?

Reply:
Done! Thanks for pointing out.